build.py

```diff
@@ -178,16 +174,13 @@ def build_package(target, build_native_package):
 
     # prepare executables
     os.chdir(os.path.join(script_path))
-    if sys.version_info < (2, 7):
-        python_executable = get_local_path('cpython',['bin','python2.7'])
-    else:
-        python_executable = sys.executable
-        if target == 'cpython':
-            log.debug('skipping cpython ... current python version {0} >= 2.7'.format(platform.python_version()))
-            # touch file to satisfy make
-            if build_native_package:
-                touch(get_package_filename(target))
-            return
+    python_executable = sys.executable
+    if target == 'cpython':
+        log.debug('skipping cpython ... current python version {0} >= 2.7'.format(platform.python_version()))
```

Comment:
might need to decide our minimum here... and update the 2.7 to reflect that...

I think we're leaning towards 3.6 at the moment...

or just remove this debug comment?   hmmm.   we'll have other things skipping before too long, so keeping it probably makes the most sense.

Reply:
leaving as-is.